docs/index.rst

```diff
@@ -6,31 +6,35 @@ Getting Started With CrateDB Cloud
 
 Welcome to CrateDB Cloud!
 
-CrateDB Cloud is a scalable SQL Platform-as-a-Service that allows for data
-ingestion, enrichment, analytics, visualization, and monitoring of massive
-amounts of machine and internet of things (IoT) data.
+The CrateDB Cloud platform is a scalable SQL cloud service that allows for
+real-time analytics and visualization of industrial internet of things (IIoT)
+data at massive scale. The cloud service is fully-managed and operated by our
```

Comment:
drop "cloud" from "the cloud service" or maybe think about using "CrateDB Cloud" instead